### ISSUE TITLE
loop_control doesn't work with include_vars

### ISSUE THREAD

**Issue by StephaneBunel** [ansible/ansible#17101]
##### ISSUE TYPE
- Bug Report
##### COMPONENT NAME

include_vars action plugin
##### ANSIBLE VERSION

```
ansible 2.1.1.0
  config file = /home/stephane/code/prettysimple/python/ansible-pyenv/prettysimplegames/ansible.cfg
  configured module search path = Default w/o overrides
```
##### STEPS TO REPRODUCE

OK (role/task/main.yml):

```
- include_vars: "{{ item }}"
  with_first_found:
    - "vars-for-{{ ansible_distribution|lower }}-{{ ansible_distribution_version|lower }}.yml"
    - "vars-for-{{ ansible_distribution|lower }}.yml"
    - "defaults.yml"
```

NOK (role/task/main.yml):

```
- include_vars: "{{ include_vars }}"
  with_first_found:
    - "vars-for-{{ ansible_distribution|lower }}-{{ ansible_distribution_version|lower }}.yml"
    - "vars-for-{{ ansible_distribution|lower }}.yml"
    - "defaults.yml"
  loop_control:
    loop_var: include_vars

TASK [ps/ubuntu-common/roles/apt : include_vars] *******************************
Traceback (most recent call last):
  File "/home/stephane/code/prettysimple/python/ansible-pyenv/local/lib/python2.7/site-packages/ansible/executor/process/result.py", line 176, in run
    loop_var = result._task.loop_control.get('loop_var') or 'item'
AttributeError: 'LoopControl' object has no attribute 'get'
```
##### RELATED PR:

https://github.com/ansible/ansible/pull/16538

**Comment by jctanner** [comment]
@StephaneBunel has this been resolved by #16538?

needs_info

**Comment by jimi-c** [comment]
The fix in #16538 did address this, and I've cherry-picked this over to stable-2.1 as well, so it will be included in 2.1.2.

If you continue seeing any problems related to this issue, or if you have any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular issue is resolved.

Thank you!
